### PR TITLE
Add `interactive_timeout` variable

### PR DESCRIPTION
**Pull request by glkappe**
https://github.com/pingcap/tidb/pull/8573
此变量在 3.0 release not 中已被提及，辛苦技术审核，帮忙添加此系统变量相关描述。

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)
 

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by TomShawn** [comment]
@lysu 请帮忙添加该变量的详细信息，谢谢！

File: system-variables.md [hunk 1]
```diff
@@ -78,6 +78,12 @@ SET  GLOBAL tidb_distsql_scan_concurrency = 10;
 >
 > `max_execution_time` 目前对所有类型的语句生效，并非只对 `SELECT` 语句生效，与 MySQL 不同（只对`SELECT` 语句生效）。实际精度在 100ms 级别，而非更准确的毫秒级别。
 
+## `interactive_timeout`
+
+- 作用域：
+- 默认值：
+-
```
Comment (lysu) [inline system-variables.md:85]:
```suggestion
- 作用域：SESSION | GLOBAL
- 默认值：28800
- 这个变量表示交互式用户会话的空闲超时，单位为秒。交互式用户会话是指使用 `CLIENT_INTERACTIVE` 选项调用 `mysql_real_connect()` API 建立的会话(比如： MySQL shell 客户端)。
```

Comment (lysu) [reply system-variables.md:85]: also, this variable is fully compatible with MySQL

https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_interactive_timeout 

@TomShawn  PTAL， thx

Comment (TomShawn) [reply system-variables.md:85]: @lysu Thanks very much!

**Pull request merged**